### ISSUE TITLE
Get an error while trying to search

### ISSUE THREAD

**Issue by Jim-Zenn** [jeffkowalski/geeknote#59]
```
geeknote dedup insurance
   : Connecting to Evernote...Downloaded metadata for 2 notes (2/2 of 20)
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/geeknote/geeknote.py", line 1233, in main
    Notes().dedup(**ARGS)
  File "/usr/local/lib/python2.7/site-packages/geeknote/geeknote.py", line 1082, in dedup
    noteId = note.title + ":" + note.contentHash
AttributeError: 'NoteMetadata' object has no attribute 'contentHash'
```

**Comment by jeffkowalski** [comment]
What version of geeknote are you using?
Please list the output of the command ```geeknote``` alone, without any arguments.
It should print something like this:
```
Version: 2.0.7
Geeknote - a command line client for Evernote.
Use geeknote --help to read documentation.
```

**Comment by Jim-Zenn** [comment]
It's 2.0.7

**Comment by jeffkowalski** [comment]
Should be fixed now in d2616d2157db0d38a7f045e961eaccbd612e6d85
